code/modules/mob/living/living.dm

```diff
@@ -635,6 +635,10 @@ default behaviour is:
 /mob/living/carbon/get_contained_external_atoms()
 	. = ..()
 	LAZYREMOVE(., get_organs())
+	//Since organs are cleared on destroy, add this separate check here
+	for(var/obj/item/organ/O in .)
+		if(!O.is_droppable())
+			. -= O
```

Comment:
Needs to be LAZYREMOVE so that `.` is nulled if the list is cleared, to line up with previous implementation.

Reply:
Fixed